pages/index.tsx

```diff
@@ -49,22 +49,23 @@ const Home: NextPage = () => {
       </Head>
 
       <main className={styles.main}>
-        <div className={styles.header}>
-          {isDark ? 
-          <Text h3 css={{cursor: "pointer"}} onClick={()=>setTheme('light')}>🌕</Text>
-          : 
-          <Text h3 css={{cursor: "pointer"}} onClick={()=>setTheme('dark')}>🌑</Text> 
-        }
-        </div>
+          <div className={styles.header}>
+            {isDark ? 
+            <Text h3 css={{cursor: "pointer"}} onClick={()=>setTheme('light')}>🌕</Text>
+            : 
+            <Text h3 css={{cursor: "pointer"}} onClick={()=>setTheme('dark')}>🌑</Text> 
+          }
+          </div>
+       <div className={styles.body}>
         <Image alt="Logo" src={Logo} />
-        <Text h2>UseThisIcon.com</Text>
-        <div className={styles["info-container"]}>
-          <Text size="1.25rem" weight="medium" css={{ textAlign: "center" }}>
-            Generate an icon that you MUST to use at some point of the day
-          </Text>
-          <Text size="1.25rem">What are you waiting for?</Text>
-        </div>
-        {currentIcon ? (
+          <Text h2>UseThisIcon.com</Text>
+          <div className={styles["info-container"]}>
+            <Text size="1.25rem" weight="medium">
+              A website that generate a radom icon per day!
+            </Text>
```

Comment:
I think that this one is the last message that we defined `Generate an icon that you MUST use at some point of the day`. Also, try to check that this works properly on mobile devices

Reply:
But that is already in the next page that you suppose to be doing, right ? Look at the Figma